plugins/@grouparoo/iterable/src/lib/export/exportProfile.ts

```diff
@@ -69,32 +69,27 @@ export const sendProfile: ExportProfilePluginMethod = async ({
       { dataFields: formattedDataFields }
     );
 
-    let currentUser = await getUser(client, email);
     if (currentEmail && currentEmail !== email) {
-      const oldUser = await getUser(client, currentEmail);
-      if (oldUser) {
-        const emailPayload = { currentEmail, newEmail: email };
-        if (newProfileProperties.userId) {
-          emailPayload["currentUserId"] = newProfileProperties.userId;
-        }
-        await client.users.updateEmail(emailPayload);
-        currentUser = oldUser;
-        currentUser.email = email;
+      const emailPayload = { currentEmail, newEmail: email };
+      if (newProfileProperties.userId) {
+        emailPayload["currentUserId"] = newProfileProperties.userId;
       }
+      await client.users.updateEmail(emailPayload);
```

Comment:
let's get a test to understand what happens here if the `old` email address is not in the system. Might have to catch.